### PR TITLE
GP2-3338 - Dashboard top 3 export plans

### PR DESCRIPTION
**Pull request by bobmeredith**
Replaces the old dashboard progress card on the domestic dashboard page, with a new card showing a list of the last 3 created export plans.
Also a bit of tidying old rubbish that's no longer relevant

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-3338
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by bobmeredith** [comment]
image

**Comment by bobmeredith** [comment]
<details>
<summary>Export plan card</summary>
image


</details>

File: domestic/templates/domestic/dashboard_cards/exportplan_progress.html [hunk 1]
```diff
@@ -1,57 +1,24 @@
...
-          {% else %}
-              <h3 class="h-s p-t-s">You've completed your export plan</h3>
-              <div class='w-1-2'>
-                  <p class='m-t-xs'>Share it with lenders and potential partners to get started on your exporting journey</p>
-              </div>
-          {% endif %}
-
-      </div>
+<div class="g-card">
+  <div class="g-card-progress {% if export_plan_progress.exportplan_completed %}g-card-progress--complete{% else %}" style="background-image: url({% static 'images/' %}{{export_plan_progress.next_section.image}})" {% endif %}">
```
Comment (froddd) [inline domestic/templates/domestic/dashboard_cards/exportplan_progress.html:9]: Is this class/background image relevant now?

File: exportplan/templates/exportplan/includes/list_card_small.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{% load content_tags %}
+{% load humanize %}
+{% comment %}
+EP Card for smaller, dashboard-style listing
+{% endcomment %}
+<article class="relative bg-white p-h-s p-v-xxs clearfix m-b-xxs">
+  <a href="{{plan.url}}" class="learn__category-link" aria-describedby="export-plan-{{ plan.url }}">
```
Comment (froddd) [inline exportplan/templates/exportplan/includes/list_card_small.html:7]: Super nit-pick: spaces inside double curlies ;)

**Review by froddd — APPROVED** [review]
Couple of nitpicks, but looks good! Excellent work on the tidy-up/code removal. Jealous.

**Pull request merged**